tools/abricate/abricate.xml

```diff
@@ -5,11 +5,10 @@
     <macros>
         <import>macros.xml</import>
     </macros>
-    <expand macro='xrefs'/>
-    <expand macro='edam_ontology'/>
-    <expand macro="requirements" />
-    <expand macro="version_command" />
-
+    <expand macro="edam_ontology"/>
```

Comment:
If we fix a linter error we should remove it from the tool's `.lint_skip` file.

Reply:
abcriacte didn't had a lint_skip file imho